gradlew

```diff
@@ -86,7 +86,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=${0##*/}
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
+DEFAULT_JVM_OPTS='"-Xmx1024m" "-Xms256m"'
```

Comment:
Isn't gradle.properties taking precedence and so this is unnecessary?

Reply:
Sadly, it is necessary due to out of memory conditions we encountered while compiling.

Reply:
I wonder if that will get blatted away the next time we update gradle 🤔

Reply:
...looks like you might be about to find out :)
https://github.com/hyperledger/besu/pull/5144
cc @usmansaleem